piicatcher/explorer/databases.py

```diff
@@ -236,7 +236,6 @@ class PostgreSQLExplorer(RelDbExplorer):
             INFORMATION_SCHEMA.COLUMNS
         WHERE
             TABLE_SCHEMA NOT IN ('information_schema', 'pg_catalog')
-            AND DATA_TYPE SIMILAR TO '%char%|%text%'
```

Comment:
What is this exactly? It limits only on columns with char or text datatype?

Reply:
yes. that was the reason `location_lat`, `location_lon` wasn't coming up, as they're saved as integer in database.
![image](https://user-images.githubusercontent.com/17861054/103284308-e5282180-4a00-11eb-9631-0ef54494af5a.png)